packages/core/admin/admin/src/components/AuthenticatedApp/tests/index.test.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { QueryClientProvider, QueryClient } from 'react-query';
+import { fetchAppInfo, fetchCurrentUserPermissions, fetchStrapiLatestRelease } from '../utils/api';
+import packageJSON from '../../../../../package.json';
+import AuthenticatedApp from '..';
+
+const strapiVersion = packageJSON.version;
+
+jest.mock('../utils/api', () => ({
+  fetchStrapiLatestRelease: jest.fn(),
+  fetchAppInfo: jest.fn(),
+  fetchCurrentUserPermissions: jest.fn(),
+}));
+
+jest.mock('../../PluginsInitializer', () => () => <div>PluginsInitializer</div>);
+// eslint-disable-next-line react/prop-types
+jest.mock('../../RBACProvider', () => ({ children }) => <div>{children}</div>);
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+    },
+  },
+});
+
+const app = (
+  <QueryClientProvider client={queryClient}>
+    <AuthenticatedApp />
+  </QueryClientProvider>
+);
+
+describe('Admin | components | AuthenticatedApp', () => {
+  beforeEach(() => {
+    jest.resetModules(); // Most important - it clears the cache
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should not crash', () => {
+    fetchStrapiLatestRelease.mockImplementation(() => Promise.resolve({ tag_name: strapiVersion }));
+    fetchAppInfo.mockImplementation(() =>
+      Promise.resolve({
+        autoReload: true,
+        communityEdition: false,
+        currentEnvironment: 'development',
+        nodeVersion: 'v14.13.1',
+        strapiVersion: '3.6.0',
+      })
+    );
+    fetchCurrentUserPermissions.mockImplementation(() => Promise.resolve([]));
+
+    expect(render(app)).toMatchInlineSnapshot(`
```

Comment:
ℹ️ `render()` returns way too much information( check the snapshot, you got a load shit of stuff 😛 )

If you want to stick with a snapshot, a more appropriate way to do would be:

```jsx
const { container } = render(app)

expect(container.firstChild).toMatchInlineSnapshot()
```